elements/pfe-tabs/test/pfe-tabs_e2e.js

```diff
@@ -1,24 +1,46 @@
 const element = require("../package.json").pfelement.elementName;
+let windowSize;
 
 describe(element, () => {
-  let snapshotOptions;
-
   before(() => {
-    browser.url(`/elements/${element}/demo`);
+    // Capture window size
+    windowSize = browser.getWindowSize();
 
-    snapshotOptions = {
-      removeElements: [
-        $("#toc"),
-        $$(".skip-e2e")
-      ]
-    };
+    browser.url(`/elements/${element}/demo/index_e2e.html`);
+    
+    browser.setWindowSize(1200, 1000);
   });
 
-  ["light", "dark", "saturated"].forEach(context => {
-    it(`should take a screenshot and compare for ${context} context`, () => {
-      $("#context").selectByAttribute("value", context);
-      browser.saveFullPageScreen(`${element}--${context}`, snapshotOptions);
-      expect(browser.checkFullPageScreen(`${element}--${context}`, snapshotOptions)).toBeLessThan(3.1);
-    });
+  after(function() {
+    browser.setWindowSize(windowSize.width, windowSize.height);
   });
+
+  if (browser.capabilities.browserName === "IE") {
```

Comment:
IE doesn't use the other background contexts - it just defaults to white anyway so not necessary to snapshot every context